docs/Onco-Query-Language.md

```diff
@@ -128,6 +145,23 @@ MDM2: AMP GAIN HOMDEL EXP > 1.5 EXP < -1.5
 TP53: AMP GAIN HOMDEL EXP > 1.5 EXP < -1.5
 ```
 
+## Merged gene tracks
+OncoPrint has functionality to merge multiple genes into a single track. This can be done by enclosing a set of genes in squared brackets, for example:
+```
+[ATM BRCA1 BRCA2]
+```
+![image of rb query](images/OQL/merged_genes_oncoprint.png)
+
+This track of merged genes can be expanded to view individual tracks:
+
+![image of rb query](images/OQL/merged_genes_oncoprint_expanded.png)
```

Comment:
Minor accessibility detail: the title texts of the two images shouldn't be identical if the information they convey isn't. Although I can't think of a situation where someone who can use the Oncoprint can't view these PNG images, which would be the target audience for the title texts.